src/Workspaces/Remote/Core/ServiceDescriptors.cs

```diff
@@ -24,6 +24,7 @@
 using Microsoft.CodeAnalysis.NavigationBar;
 using Microsoft.CodeAnalysis.Rename;
 using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.CodeAnalysis.SolutionEvents;
```

Comment:
>SolutionEvents [](http://example.com/codeflow?start=29&length=14)

Do we need a new namespace? Workspace event related types are in Microsoft.CodeAnalysis

Reply:
I'd like a new namespace here to keep this isolated.

Reply:
ok